METIS/tests/rescale_psf.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from scipy.ndimage import zoom
+from astropy.io import fits
+
+
+def rescale_image(old_hdu, new_cdelt, max_size=512):
+    scale_factor = old_hdu.header["CDELT1"] / new_cdelt
+    image = zoom(old_hdu.data, scale_factor)
+    image /= image.sum()
+    y, x = np.divmod(np.argmax(image), image.shape[1])
+    r = max_size // 2
```

Comment:
Using `np.unravel_index` might be more explicit:
```
y, x = np.unravel_index(np.argmax(image), image.shape)
```
Code would be a bit less cryptic if you used longer variabel names...
I'm slightly uneasy about using argmax for the centre - it seems unlikely, but there might be cases where we're dealing with wobbly PSFs (well, the HCI PSFs do wobble).

Reply:
Yep. Removing the K-band PSF is why there is no more jump. The cause remains a mystery.

Re wobbly PSFs. I'm happy to use a different method to determine the centre. This was just a super quick hack to get the current PSF reformatted so that we could include it in the release, purely due to the time constraints,